tensorflow/python/ops/init_ops.py

```diff
@@ -138,8 +141,8 @@ def constant_initializer(value=0, dtype=dtypes.float32):
     >>> with tf.Session():
     >>>   x = tf.get_variable('x', shape=[2, 3], initializer=init)
 
-    ValueError: Too many elements provided. Needed at most 6, but received 8
-    ```
+    ValueError: Too many elements provided. Needed at most 6, but received 8.
```

Comment:
So what was it that was causing the formatting bug? The lack of a period at the end of the sentence or the extra space in front of the ```? 

Can you run the script locally and verify that the docs are generated correctly?